lib/Doctrine/ORM/Cache/Lock.php

```diff
@@ -19,20 +16,13 @@ class Lock
      */
     public $time;
 
-    /**
-     * @param string $value
-     * @param int    $time
-     */
-    public function __construct($value, $time = null)
+    public function __construct(string $value, int $time = null)
     {
         $this->value = $value;
         $this->time  = $time ?: time();
     }
 
-    /**
-     * @return \Doctrine\ORM\Cache\Lock
-     */
-    public static function createLockRead()
+    public static function createLockRead(): Lock
```

Comment:
Let's not change public API just yet, will be done later everywhere + needs to be documented in UPGRADING.